README.md

```diff
@@ -1 +1,52 @@
-# Hello-world4
\ No newline at end of file
+# Hello-world4
+
+> In this project, repository was set to master all of the tools and best practices that I learnt about(Documenting repo in a professional way,Git & GitHub & GitHub Flow and Linters) in previous steps.
+
+
```

Comment:
- [ ] Kindly make sure the description of the project goes here. you can use [this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Estimated%20time%3A%200.5h-,Description,-In%20this%20project) and change it a little bit